provider/gce/credentials.go

```diff
@@ -52,7 +52,7 @@ func (environProviderCredentials) CredentialSchemas() map[cloud.AuthType]cloud.C
 		cloud.JSONFileAuthType: {{
 			Name: credAttrFile,
 			CredentialAttr: cloud.CredentialAttr{
-				Description: "path to the .json file containing your Google Compute Engine project credentials",
+				Description: "path to a service account key",
```

Comment:
can we keep the bit about it being a .json file, eg

"path to the .json file containing a service account key for your project"

Reply:
Maybe we can add a discourse link in the description as well

Reply:
I've added a link to discourse thread that I'll flesh out before 2.6 is released.